numpy/core/tests/_runtime_detect.py

```diff
@@ -0,0 +1,10 @@
+def main():
```

Comment:
It would be better to create it as temp file at runtime instead within `setup_method`, it will improves the readability.